QgisModelBaker/gui/generate_project.py

```diff
@@ -274,9 +274,10 @@ def accepted(self, edited_command=None):
                         return
 
                 if self.ili_file_line_edit.text().strip() and \
-                        self.ili_file_line_edit.validator().validate(configuration.ilifile, 0)[0] != QValidator.Acceptable:
+                    self.ili_file_line_edit.validator().validate(configuration.ilifile, 0)[0] != QValidator.Acceptable:
+
                     self.txtStdout.setText(
-                        self.tr('Please set a valid INTERLIS file before creating the project.'))
+                        self.tr('Please set a valid INTERLIS file before creating the project. {}').format(self.ili_file_line_edit.validator().error))
```

Comment:
This will print `None` if unset, is that something to take care of?

Reply:
I usually prefer `None` over `''` and in this case it's not important since we do not print this when `error` is empty. But maybe for future use of this `''` would be better in case you attach it to a string...